### ISSUE TITLE
Scatter plot - Max Interpolation Width

### ISSUE THREAD

**Issue by JNo1492** [ghtmtt/DataPlotly#350]
I've been using this tool to create cross section profiles, which is an incredible development:
![image](https://github.com/user-attachments/assets/832228d4-7cdc-4299-a220-cc0a5ace3dde)

As you can see, I am plotting the values on the scatter plot as points because if I use lines - the line would join the gap where there is no data for that depth unit. However, now there are visible gaps between points where there is a large jump in vertical change.

Basically, it would be good to be able to set a max interpolation distance, so I could make it so the points only connect with lines where the gap between points is less than for example, 10m along the x-axis.
